packages/ember/src/components/loading-indicator.gts

```diff
@@ -0,0 +1,49 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export const typeBorder = 'border';
+export const typeGrowing = 'growing';
+
+interface LoadingIndicatorSignature {
+  Element: HTMLDivElement;
+  Args: {
+    label?: string;
+    showLabel?: boolean;
+    type?: typeof typeBorder | typeof typeGrowing;
+  };
+}
+
+export default class LoadingIndicatorComponent extends Component<LoadingIndicatorSignature> {
+  get typeClass() {
+    const { type } = this.args;
+    assert(
+      'type must be either `border` or `growing`',
+      type === undefined || type === typeBorder || type === typeGrowing,
+    );
+
+    if (type === typeGrowing) {
+      return 'spinner-growing';
```

Comment:
```suggestion
      return 'spinner-grow';
```

Reply:
Don't know how I missed that! I blame Copilot 😄